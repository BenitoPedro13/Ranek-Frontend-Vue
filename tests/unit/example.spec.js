import pizzas from './data.json'

test('the pizzaa data is correct', () => {
  expect(pizzas).toMatchSnapshot()
  expect(pizzas).toHaveLength(4)
  expect(pizzas.map(pizza => pizza.name)).toEqual([
    'New York Pizza',
    'San Francisco Pizza',
    'Dallas Pizza',
    'Chicago Pizza'
  ])
})

for(let i = 0; i < pizzas.length; i++){
  test()
}
