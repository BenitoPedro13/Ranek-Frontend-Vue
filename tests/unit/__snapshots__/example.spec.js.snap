// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the pizzaa data is correct 1`] = `
Array [
  Object {
    "desc": "New York-style pizza has slices that are large and wide with a thin crust tha is foldable yet crispy.",
    "id": 1,
    "image": "/images qny-pizza.jpg",
    "name": "New York Pizza",
    "price": 8,
  },
  Object {
    "desc": "San Francisco-style pizza has slices that are large and wide with a thin crust tha is foldable yet crispy.",
    "id": 2,
    "image": "/images/sf-pizza.jpg",
    "name": "San Francisco Pizza",
    "price": 10,
  },
  Object {
    "desc": "Dallas-style pizza has slices that are large and wide with a thin crust tha is foldable yet crispy.",
    "id": 3,
    "image": "/images/da-pizza.jpg",
    "name": "Dallas Pizza",
    "price": 7,
  },
  Object {
    "desc": "Chicago-style pizza has slices that are large and wide with a thin crust tha is foldable yet crispy.",
    "id": 4,
    "image": "/images/ch-pizza.jpg",
    "name": "Chicago Pizza",
    "price": 9,
  },
]
`;
